zcash_client_backend/Cargo.toml

```diff
@@ -247,6 +249,11 @@ unstable-serialization = ["dep:byteorder"]
 ## Exposes the [`data_api::scanning::spanning_tree`] module.
 unstable-spanning-tree = []
 
+## feature to allow building for wasm using wasm-bindgen
+wasm-bindgen = [
+    "time/wasm-bindgen",
+]
+
```

Comment:
Is this necessary or only a convenience? AFAICT downstream WASM crates can themselves add a dependency on `time` with the necessary feature flag enabled, without us needing to re-export the feature flag. I'm reluctant to re-export uncommon feature flags from crates we do not control.
```suggestion
```